runtime/onert/core/include/util/ShapeInference.h

```diff
@@ -47,7 +47,7 @@ ir::Shape inferArgMaxShape(const ir::Shape &input_shape, int axis, int rank);
 ir::Shape inferBatchMatMulShape(const ir::Shape &lhs_shape, const ir::Shape &rhs_shape,
                                 const ir::operation::BatchMatMul::Param &param);
 
-ir::Shape inferBroadcastToShape(const ir::Shape wshape, const int32_t *shape_buffer);
+ir::Shape inferBroadcastToShape(const ir::Shape shp_shape, const int32_t *shape_buf);
```

Comment:
I am not sure of the meaning of `shp_shape`.
how about something like `src_shape` and `dest_shape_buf`?

Reply:
@hyunsik-yoon 
That's the confusing part.

shp_shape means that  `SHAPE` tensor's shape.
The input tensor's name is 'SHAPE', so shp_shape means 'shape of the `SHAPE` tensor' and shape_buf means 'buffer of the `SHAPE` tensor'. 

I followed the other function's cases.
I think src_shape and dest_shape_buf doesn't fit the underlying meaning too. it's no source and dest, it's actually shape and buffer of the shape tensor..
May I add some explanation then?

Reply:
I see. Let's just leave as you named. :-)